roles/cifmw_cephadm/tasks/post.yml

```diff
@@ -17,6 +17,27 @@
 - name: Get ceph_cli
   ansible.builtin.include_tasks: ceph_cli.yml
 
+- name: Configure ceph object store to use external ceph object gateway
+  cifmw.general.ci_script:
+    extra_args:
+      KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    output_dir: "/home/zuul/ci-framework-data/artifacts"
+    script: |-
+      oc rsh openstackclient openstack service create --name swift --description 'OpenStack Object Storage' object-store
+      oc rsh openstackclient openstack user create --project service --password {{ cifmw_ceph_rgw_keystone_psw }}  swift
+      oc rsh openstackclient openstack role create swiftoperator
+      oc rsh openstackclient openstack role create ResellerAdmin
+      oc rsh openstackclient openstack role add --user swift --project service member
+      oc rsh openstackclient openstack role add --user swift --project service admin
+      oc rsh openstackclient openstack endpoint create --region regionOne object-store public http://{{ cifmw_cephadm_vip }}:8080/swift/v1/AUTH_%\(tenant_id\)s
+      oc rsh openstackclient openstack endpoint create --region regionOne object-store internal http://{{ cifmw_cephadm_vip }}:8080/swift/v1/AUTH_%\(tenant_id\)s
+      oc rsh openstackclient openstack role add --project admin --user admin swiftoperator
+  delegate_to: localhost
+  when:
+    - cifmw_ceph_daemons_layout.rgw_enabled | bool
```

Comment:
@fultonj the zuul job podified-multinode-hci-deployment-crc fails to identify sruct 'cifmw_ceph_daemons_layout' which is supposed to be passed via ceph_overrides.

Reply:
and you probably need a `| default('true')` to the boolean in case no overrides are passed

Reply:
@fmount ack will do that.